transaction_integration_test.go

```diff
@@ -725,3 +726,185 @@ func TestTransactionCreateWithCustomFields(t *testing.T) {
 		t.Fatalf("Returned custom fields doesn't match input, got %q, want %q", x, customFields)
 	}
 }
+
+func TestHoldInEscrowOnCreate(t *testing.T) {
+	testSubMerchantAccountId := getSubMerchantAccount(t)
+	amount := NewDecimal(6200, 2)
+	txn, err := testGateway.Transaction().Create(&Transaction{
+		Type:   "sale",
+		Amount: amount,
+		CreditCard: &CreditCard{
+			Number:         testCreditCards["visa"].Number,
+			ExpirationDate: "05/14",
+		},
```

Comment:
I noticed you pulled out some code into functions at the end to DRY the code. Another thing you can do is instead of defining a credit card for each transaction, use `PaymentMethodNonce: FakeNonceTransactable`.